api/src/modules/scenarios/wdpa-arena-calculation.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { pick } from 'lodash';
+import { CreateScenarioDTO } from './dto/create.scenario.dto';
+import { UpdateScenarioDTO } from './dto/update.scenario.dto';
+import { Scenario } from './scenario.api.entity';
+
+type InputChange = CreateScenarioDTO | UpdateScenarioDTO;
+
+@Injectable()
+export class WdpaArenaCalculationService {
```

Comment:
after having squinted blankly at this class name for a while, I think I was trying to read some meaning which is not there - I suppose this is a gladiator misspelling for the name below, right?

```suggestion
export class WdpaAreaCalculationService {
```

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/162/commits/87c647950b6545a6abaff37d4f154b5347f6344a